includes/definitions/discovery/poweralert.yaml

```diff
@@ -35,6 +43,13 @@ modules:
                     index: 'tlpPduCircuitTotalCurrent.{{ $index }}'
                     divisor: 100
                     descr: "Current"
+                -   
```

Comment:
This is a duplicate of the existing tlpPduCircuitTotalCurrent sensor.  I suggest you delete the previous one, which must have had the wrong index.

Reply:
This was just recently merged into master: https://github.com/librenms/librenms/pull/13340/files#diff-46bf9d59eb5cac5439a3ab8e7b552e949981e045e3cc30ab60d14c571be1545eR30 

If I deleted the previous sensor wouldn't that affect @arjitc PR? I'm not sure why the index worked on his tests so recently, it would be interesting to know what device he was using to test.

Reply:
Hi @murrant  / @drommc

I'm not too sure which model we use exactly, but they were recently upgraded to the same LXCARD 20 version and tlpPduCircuitTotalCurrent should still work (basically tlpPDUCircuitTotalCurrent.1.1 or tlpPDUCircuitTotalCurrent.1.2) etc

I got the OIDs from https://assets.tripplite.com/technical-bulletins/supported-snmp-oids-technical-bulletin-en.pdf